### ISSUE TITLE
Only first row of data is applied when syncing to nested components

### ISSUE THREAD

**Issue by maracht** [DWilliames/google-sheets-sync-figma#114]
**Describe the bug**
Only the first row data is applied when using Google Sheets Sync on **nested components**.
![image](https://user-images.githubusercontent.com/68451624/172527102-601f3021-c626-428b-8d3d-39d2ca089c67.png)
![image](https://user-images.githubusercontent.com/68451624/172527146-c1b6d92b-3a41-4a87-b953-076aef0a9816.png)
![image](https://user-images.githubusercontent.com/68451624/172527797-5707afb2-cf37-4639-b5f6-d7c9f51392da.png)

**To Reproduce**
Run sync on the component
Figma file: https://www.figma.com/file/6Mv5HAu1kPMlq3bgMgfmHI/For-Github-Big-report?node-id=0%3A1
Google sheet: https://docs.google.com/spreadsheets/d/1xLfR8WaTFky3RoUogJDMI2Flr_flbCRAwYbb8JNbNhw/edit?usp=sharing

**Expected behavior**
All data should be applied, instead of just the first row of the data
![image](https://user-images.githubusercontent.com/68451624/172527615-d711cb4e-cfbd-44e5-a22f-b8facf7d58d9.png)

**Comment by r-carlander** [comment]
Hello,
If you name your layers "#article_image.1", "#article_image.2", "#article_image.3" etc, it will work. Not a perfect solution, but it works.
![Screenshot 2022-06-08 at 09 18 22](https://user-images.githubusercontent.com/3692747/172555734-3d24b8c1-9d39-4bd4-a492-fa6b6ef0832e.png)

**Comment by maracht** [comment]
@r-carlander Thanks for the workaround. I'll try this in the meanwhile.

In my opinion, naming layers is not ideal as it takes some time. 

On the other hand, if this is a bug, fixing it will save people a lot of time :)

**Comment by r-carlander** [comment]
@maracht I agree! But it seems like @DWilliames is very busy, he doesn't check in here very often. So the workaround is better than nothing I guess 😅

**Comment by jonleung** [comment]
Thanks @r-carlander this worked for me, appreciate the heads up on this!

I do agree with @maracht that it would be ideal for this to work with Nested components as well.

Thanks to you both, and especially to @DWilliames for creating this! Would love to have this functionality @DWilliames : )

**Comment by jonleung** [comment]
This was helpful though to work with @r-carlander's solution: https://help.figma.com/hc/en-us/articles/360039958934-Rename-Layers and then using "Shift + Cmd" Click to click directly on the sub component.
